_content/articles/plant-reliefs-at-borobudur_metusala-et-al.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "The identification of plant reliefs in the Lalitavistara story of Borobudur temple, Central Java, Indonesia"
+authors:
+  - "Destario Metusala"
+  - "Fauzuah Fauziah"
+  - "Dewi Ayu Lestari"
+  - "Janis Damaiyani and others" 
+external_url: "https://smujo.id/biodiv/article/view/5530/3877"
+source_url: "https://doi.org/10.13057/biodiv/d210549"
+drive_links:
+  - "https://drive.google.com/drive/folders/10FhXe3dtyCmt_uQt0VXD8hpMJaDCjS2T"
```

Comment:
Oops! One mistake. This is the link to the entire Google Drive folder. The link just to this one document is https://drive.google.com/file/d/1OOaxIWCbDcVwrUYaLt2F8u6q3FXcYzSf/view?usp=drivesdk

I'll go ahead and fix that, no worries.